### DISCUSSION TITLE
Replace the game details box on hubs with a field for a short text blurb.

### DISCUSSION THREAD

**Discussion by WandKnight** [RetroAchievements/RAWeb#1046] [Ideas]
The current fields don't really offer any room for useful information that can't be gleamed from the title of the hub itself, and oftentimes they're just left blank. I'd like to see these replaced by a single text box that developers can use to input a short explanation of the purpose of the hub.

**Comment by Krylan** [comment]
Just on a side note: I think we should completely change the way, how hubs are working. Right now, they are a game records, sharing the same data structure and even the same URL schema. In my opinion, we should treat them as different type of entities, with text description in question, rather than other data (like developer, genre, release date etc., which are specific to a game). It would of course require doing some work, but I think it would be beneficial in managing that in the future.

**Comment by luchaos** [comment]
@Krylan v2 will do just that. As for v1 we might want to think about a common game/hub field to allow for a "short text blurb". Otherwise we should just close this issue.
